app/src/main/res/values-v27/styles.xml

```diff
@@ -14,4 +14,18 @@
         <item name="android:windowLightStatusBar">@bool/isLightMode</item>
         <item name="android:windowLightNavigationBar">@bool/isLightMode</item>
     </style>
+
+    <style name="Theme.HomeAssistant" parent="Theme.MaterialComponents.DayNight.NoActionBar">
+        <item name="colorPrimary">@color/colorPrimary</item>
+        <item name="colorPrimaryDark">@color/colorPrimaryDark</item>
+        <item name="colorAccent">@color/colorAccent</item>
+        <item name="colorSecondary">@color/colorAccent</item>
+        <item name="colorSecondaryVariant">@color/colorAccent</item>
+        <item name="colorButtonNormal">@color/colorPrimary</item>
+        <item name="android:colorBackground">@color/colorBackground</item>
+        <item name="android:navigationBarColor">@color/colorPrimaryDark</item>
+        <item name="alertDialogTheme">@style/Theme.HomeAssistant.Dialog.Alert</item>
+        <item name="actionBarPopupTheme">@style/ThemeOverlay.MaterialComponents.Dark</item>
+        <item name="android:windowLayoutInDisplayCutoutMode">shortEdges</item>
```

Comment:
Is `shortEdges` the right value here (rendering content behind cutouts)? The app doesn't enable rendering behind cutouts for the webview, does it? Even without enabling full screen mode, I now see a difference in landscape mode (there is a cutout in the bottom left):
|Stable app|This PR|
|-----|-----|
|![image](https://user-images.githubusercontent.com/8148535/189988206-dc4f67d4-12dc-4314-9dcd-0196aa177a54.png)|![image](https://user-images.githubusercontent.com/8148535/189988185-339da907-efda-40b9-9e5a-07971e68258c.png)|

Reply:
I wasn't sure about this one honestly, I just know that if we don't add this line then the theme does not push up to the top and it looked odd so wanted to fix that :)

What option do you think works best?

Reply:
It's a black bar (system hides UI) or white/dark grey bar (system shows UI: window background) in the area for me, so I don't quite follow?

Reply:
So if we don't use `shortEdges` or use the new theme line item I end up seeing a black bar where the system bar which looks odd

![image](https://user-images.githubusercontent.com/1634145/189991298-08678ee2-7cf3-48b4-8b55-517de6dde02a.png)

With the new line and set to `shortEdges` it looks like:

![image](https://user-images.githubusercontent.com/1634145/189991420-6976b060-9d9a-4f81-8b70-1dda446856b3.png)

I am not sure if we can get away with the new landscape behavior when a cut out is used, that seems to be new behavior

Reply:
It's black + dark blue vs dark grey + dark blue in dark mode. They're closer but it's not a major difference in my opinion (especially on a phone with lower brightness, where it's barely noticeable). 

The difference is more obvious in light mode: you get a black or white bar + HA blue toolbar. No matter which option you choose, it looks odd. So why change it?

Reply:
ok will leave the theme as is then :)